src/manifold/src/quickhull.cpp

```diff
@@ -801,11 +816,22 @@ void QuickHull::reclaimToIndexVectorPool(std::unique_ptr<Vec<size_t>>& ptr) {
   indexVectorPool.reclaim(ptr);
 }
 
-bool QuickHull::addPointToFace(typename MeshBuilder::Face& f,
-                               size_t pointIndex) {
+void QuickHull::addPointToFace(typename MeshBuilder::Face& f, size_t pointIndex,
+                               int& pointMutex) {
   const double D =
       getSignedDistanceToPlane(originalVertexData[pointIndex], f.P);
   if (D > 0 && D * D > epsilonSquared * f.P.sqrNLength) {
+    // Only adds point to 1 face
+
+    // Use when ExecutionPolicy::Par
+    // if (AtomicAdd(pointMutex,1)!=0)
+    // return true;
+
+    // For ExecutionPolicy::Seq
+    pointMutex = 1;
```

Comment:
what is this `pointMutex` thing?

Reply:
it's just a variable I am using to check if that particular point has been assigned to a face yet, I initially planned to have a lock for it, forgot to change the name. I'll change the name.